ghost/admin/app/services/feature.js

```diff
@@ -97,9 +119,41 @@ export default class FeatureService extends Service {
         }
     }
 
+    @computed()
```

Comment:
Nit: No need for `@computed` here

Reply:
Haha, yeah, no. Thats silly. I'll fix that.